tests/subsys/modem/backends/uart/testcase.yaml

```diff
@@ -1,21 +1,21 @@
 # Copyright (c) 2023 Trackunit Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+common:
+  harness: ztest
+  harness_config:
+    fixture: gpio_loopback
+  platform_allow:
+    - b_u585i_iot02a
+    - nrf5340dk_nrf5340_cpuapp
+
 tests:
   modem.backends.uart.async:
-    tags: modem_backend
-    harness: ztest
-    platform_allow:
-      - b_u585i_iot02a
-      - nrf5340dk_nrf5340_cpuapp
+    tags: modem_backend_uart_async
```

Comment:
do you need this tag somewhere? the story with tags is a bit like with fixtures, there are no written rules for them. They can be used for filtering tests in twister. Will ones you propose bring any value? The name is quite lengthy and looks very specific. If it is the single place it exist it won't be that useful.